test-suite/js/src/test/scala/org/scalajs/testsuite/utils/JUnitTests.scala

```diff
@@ -0,0 +1,57 @@
+package org.scalajs.testsuite.utils
+
+import com.novocode.junit.{RichLogger, RunSettings}
+import org.scalajs.junit._
+import org.scalajs.testinterface.ScalaJSClassLoader
+import sbt.testing.{Event, EventHandler, Logger, TaskDef}
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExport
+
+@JSExport
+object JUnitTests {
+  @JSExport
+  def executeAll(): Unit = {
+    val classLoader = new ScalaJSClassLoader(js.Dynamic.global)
+
+    val runSettings = new RunSettings(color = false, decodeScalaNames = true,
+        quiet = false, verbose = true, logAssert = true, ignoreRunners = "",
+        logExceptionClass = true)
+
+    val masterRunner = new JUnitMasterRunner(Array(), Array(), classLoader,
+        runSettings)
+
+    val loggers = Array[Logger](JSConsoleLogger)
+
+    for (testDef <- TestDetector.getDetectedTests()) {
+      testDef match {
+        case testDef: JUnitTestBootstrapper =>
+          val runner = new JUnitSlaveRunner(Array(), Array(), classLoader,
+              (m: String) => masterRunner.receiveMessage(m), runSettings)
+          val name = testDef.toString.split("\\$scalajs\\$junit\\$bootstrapper").head
+          val eventHandler = new EventHandler {
+            def handle(event: Event): Unit = ()
+          }
+          val richLogger = new RichLogger(loggers, runSettings, name)
+          val taskDef = new TaskDef(name, JUnitFingerprint, false, Array())
```

Comment:
Instead of exposing `JUnitFingerprint` in an ad hoc way, could we have `val framework = new JUnitFramework` and then use `framework.fingerprints.head`, like we do in `TestRunner`?